states/corvee.mod.py

```diff
@@ -20,7 +20,15 @@ def get_names():
             return []
         if not response.ok:
             return []
-        return response.json()["selected"]
+        return response.json()
+
+    def is_selected(self, names, name):
```

Comment:
This can be replaced with `if name in names`?

Reply:
Je hebt helemaal gelijk, is aangepast nu!